--- ./djangoproject/settings/common.py  2022-02-03 17:52:24.665503041 +0200
+++ ./djangoproject/settings/common.py.new      2022-02-03 17:52:45.965457510 +0200
@@ -40,7 +40,10 @@
         'PASSWORD': SECRETS.get('db_password', ''),
         'PORT': SECRETS.get('db_port', ''),
         'OPTIONS': {
-            'sslmode': 'disable',
+            'sslmode': 'verify-full',
+            'sslcert': '/app/blog/ssl/acra-client.crt',
+            'sslkey': '/app/blog/ssl/acra-client.key',
+            'sslrootcert': '/app/blog/ssl/root.crt',
         },
     },
     'trac': {
@@ -51,7 +54,10 @@
         'PASSWORD': SECRETS.get('trac_db_password', ''),
         'PORT': SECRETS.get('trac_db_port', ''),
         'OPTIONS': {
-            'sslmode': 'disable',
+            'sslmode': 'verify-full',
+            'sslcert': '/app/blog/ssl/acra-client.crt',
+            'sslkey': '/app/blog/ssl/acra-client.key',
+            'sslrootcert': '/app/blog/ssl/root.crt',
         },
     }
 }
