--- ./dev.py.old	2018-12-18 17:40:35.000000000 +0200
+++ ./dev.py.new	2018-12-19 15:12:30.000000000 +0200
@@ -1,10 +1,10 @@
 from .common import *  # noqa
 
 ALLOWED_HOSTS = [
-    'www.djangoproject.localhost',
-    'djangoproject.localhost',
-    'docs.djangoproject.localhost',
-    'dashboard.djangoproject.localhost',
+    'www.djangoproject.example',
+    'djangoproject.example',
+    'docs.djangoproject.example',
+    'dashboard.djangoproject.example',
 ] + SECRETS.get('allowed_hosts', [])
 
 DEBUG = True
@@ -32,7 +32,7 @@
 
 # django-hosts settings
 
-PARENT_HOST = 'djangoproject.localhost:8000'
+PARENT_HOST = 'djangoproject.example:8000'
 
 # django-push settings
 
