diff --git a/examples/python/common.py b/examples/python/common.py
index 87cafac..af89c1d 100644
--- a/common.py
+++ b/common.py
@@ -74,7 +74,7 @@ def get_engine(db_host, db_port, db_user, db_password, db_name, is_mysql=False,
     if not (is_mysql or is_postgresql):
         is_postgresql = True
     if is_postgresql:
-        driver = 'postgresql'
+        driver = 'cockroachdb'
         if with_tls:
             ssl_args = {
                 'sslmode': sslmode,
